crates/proto/src/dnssec/nsec3.rs

```diff
@@ -118,6 +118,10 @@ pub(super) fn verify_nsec3(
 ) -> Proof {
     debug_assert!(!nsec3s.is_empty()); // checked in the caller
 
+    let Some(soa) = soa else {
+        return nsec3_yield(Proof::Bogus, query, "SOA name not present");
```

Comment:
nit: in `dnssec/handle.rs` on L1595 in this commit there's a similar bogus nsec3 yield but it uses "missing SOA record" as the msg. Should these two `nsec3_yield` call-sites sites use the same msg wording?

Reply:
Ah! I think this falls away in 93d6b74320580d24114cff3a423298f73bc90781